### PR TITLE
Table

### PR DESCRIPTION
**Pull request by Angeleeloo**

### CODE REVIEW

File: table.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Tables</title>
+</head>
+<body>
+<table border="1">
```
Comment (kindofone) [inline table.html]: The `<table>` node should be indented one tab in, as it's a child of the `<body>` tab.

File: table.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <thead>
+    <tr>
```
Comment (kindofone) [inline table.html]: The `<tr>` element should be indented one more tab in.

File: table.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <td>row 1, cell 1</td>
+        <td>row 1, cell 2</td>
+        <td>row 1, cell 3</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
```
Comment (kindofone) [inline table.html]: The `<tr>` element should be indented one more tab in.

File: table.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <td>row 2, cell 1</td>
+        <td>row 2, cell 2</td>
+        <td>row 2, cell 3</td>
+    </tr>
+    </tbody>
+    <tfoot>
+    <tr>
```
Comment (kindofone) [inline table.html]: The `<tr>` element should be indented one more tab in.

**Comment by Angeleeloo** [comment]
Hi @kindofone , did all the above except couldn't merge to master due to conflicts. Forgot how to fix that...

**Pull request merged**